data/hai/hai reveal 5 philosophers.txt

```diff
@@ -343,7 +406,13 @@ conversation "giti tripped an alert"
 	label talk
 	`	The man, whose Republic Intelligence uniform is labeled with the name "Agent Kopkow", nods very firmly and gestures for you to take one of the only seats in the room. He takes the one opposite and Giti sits at her desk. The three agents remain standing.`
 	`	You've had some encounters with Republic Intelligence, but you know what everyone knows. They're the top police force of the Republic, dealing with high-profile civilian crimes that local authorities can't handle: murder investigations, interplanetary kidnappings, and the like. They also gather intelligence throughout human space, and possibly beyond, reporting directly to the Republic Parliament... which mostly mean it's beholden to the interests of Parliament.`
-	`	"Now Captain, your friend Giti here has tripped over a flag buried in our systems that marks what you're looking into as an item of particular and specific interest to Syndicate Internal Affairs. In fact, it is an interest which they have exclusive sole jurisdiction over, even over and above us." He looks at you meaningfully. "That requires an act of parliament in a closed session, which has not occurred for a significant length of time, if you catch my drift."`
+	choice
+		`	"What's the problem, agent?"`
+			goto uncomplaining
+		`	"I'm on an urgent mission that is critical for three governments. Don't bar my way."`
+	`	The agents smile and Kopkow says, "We don't respond well to threats."`
```

Comment:
```suggestion
		`	"What's the problem?"`
			goto uncomplaining
		`	"I'm on an urgent mission that is critical for three governments. Don't bar my way."`
	`	Uncannily, the agents all smirk at you at the same time. Kopkow responds flatly, "We don't respond well to threats."`
```
The player acts too casually in a moment where they had the option to be killed just before.